src/SFA.DAS.Payments.MatchedLearner.AcceptanceTests/TestClient.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using SFA.DAS.Payments.MatchedLearner.Api;
+using SFA.DAS.Payments.MatchedLearner.Types;
+
+namespace SFA.DAS.Payments.MatchedLearner.AcceptanceTests
+{
+    public class CustomWebApplicationFactory<TStartup> : WebApplicationFactory<TStartup> where TStartup : class
+    {
+        protected override void ConfigureWebHost(IWebHostBuilder builder)
+        {
+            Environment.SetEnvironmentVariable("APPSETTING_Environment", TestConfiguration.Environment);
+
+            builder.ConfigureAppConfiguration(config =>
+            {
+                config.Sources.Clear();
+                config.SetBasePath(Directory.GetCurrentDirectory());
+                config.AddJsonFile("appSettings.json", optional: false, reloadOnChange: false);
+                config.AddEnvironmentVariables();
+            });
+        }
+    }
+
+    public class TestClient
+    {
+        private static HttpClient _client = new HttpClient();
+        private readonly string _url;
+
+        public TestClient()
+        {
+            _url = TestConfiguration.TargetUrl;
+
+            if (!string.IsNullOrEmpty(_url)) return;
+
+            _client = new CustomWebApplicationFactory<Startup>().CreateClient(new WebApplicationFactoryClientOptions { BaseAddress = new Uri("https://localhost:44300") });
```

Comment:
There's no reason to have the options. Will work just fine as: 
`new CustomWebApplicationFactory<Startup>().CreateClient();`

Reply:
without the option the base Url is not getting set

Reply:
No need for it to be set. It works fine without it

Reply:
removed